toolkit/scripts/sodiff/mariner-sodiff.sh

```diff
@@ -2,11 +2,36 @@
 
 # Required binaries:
```

Comment:
Can you take a look at the grep errors that I think are generated from this file?
```
grep: invalid option -- '['
Usage: grep [OPTION]... PATTERNS [FILE]...
Try 'grep --help' for more information.
```
https://dev.azure.com/mariner-org/mariner/_build/results?buildId=455863&view=logs&j=2ccbf1db-6c73-5af8-5ba2-cf05469dcf51&t=62118ca4-5f53-54b3-7ec9-8ef4ab0e9237